include/gatekeeper_net.h

```diff
@@ -322,6 +321,9 @@ struct net_config {
 	 */
 	unsigned int         num_attempts_link_get;
 
+	/* The maximum packet lifetime. */
+	uint8_t              ipv6_default_hop_limits;
```

Comment:
This parameter should be part of `struct gatekeeper_if` instead of `struct net_config` because the front and back networks follow different rules.

Reply:
Good point! Done!